tests/app/confirmation_page/test_confirmation_page.py

```diff
@@ -15,7 +15,7 @@ def test_get_next_location_confirmation(self):
         answer_store = AnswerStore()
         answer_store.add(answer)
 
-        navigator = Navigator(load_schema_file("0_rogue_one.json"), answer_store)
+        navigator = Navigator(load_schema_file("0_rogue_one.json"), {}, answer_store)
```

Comment:
I think that while there is an element of functionality that exists in Navigator that works without both metadata and and AnswerStore we should support them being optional. I'm not convinced there's more logic to test as when optional they default to `{}` and an `AnswerStore` instance, if they were mandatory should still test with `{}`and an empty `AnswerStore`  instance.